ProjectLighthouse.Servers.GameServer/Controllers/Login/LoginController.cs

```diff
@@ -72,12 +70,12 @@ public async Task<IActionResult> Login()
         switch (npTicket.Platform)
         {
             case Platform.RPCS3:
-                user = await this.database.Users.FirstOrDefaultAsync(u => u.LinkedRpcnId == npTicket.UserId);
+                user = await database.Users.FirstOrDefaultAsync(u => u.LinkedRpcnId == npTicket.UserId);
```

Comment:
```suggestion
                user = await this.database.Users.FirstOrDefaultAsync(u => u.LinkedRpcnId == npTicket.UserId);
```
I'm down to ditch this style in the future but for now lets just match the rest of the codebase